### ISSUE TITLE
WCAG: Providers map accessibility 

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2171]
page: https://stadjerspas.gemeente.groningen.nl/providers?show_map=1

The map is covered by the exception and does not need to be made accessible unless the results can also be read as plain text on the page. This is not the case here.

The utility software user cannot use this page. The data entered in the left column shows results on the map. These results are not available as plain text.

Show the search results as text in, for example, the left column under the filters. For an accessible example, see how Google Maps displays search results in text.

## Temporary solution possible:
Add a button under filters that bring you to the list page
Here:
![Screenshot 2021-03-19 at 11 13 53](https://user-images.githubusercontent.com/33207036/111765536-c1a8a080-88a4-11eb-9e0f-6dc6e16f9553.png)
